src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -145,6 +145,8 @@ internal static class IDEDiagnosticIds
         public const string UnboundIdentifierId = "IDE1007";
         public const string UnboundConstructorId = "IDE1008";
 
+        public const string ConvertTypeOfToNameOfDiagnosticId = "IDE1009";
```

Comment:
Move this above the `// Analyzer error Ids` comment

```suggestion
        public const string ConvertTypeOfToNameOfDiagnosticId = "IDE0081";
```

Reply:
done.